David-Lectamo/tarea-4/caculate.js

```diff
@@ -0,0 +1,229 @@
+let operating1;
```

Comment:
@L3C74M0 , research about the use of const, let and var and refactor accordingly 

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/